### DISCUSSION TITLE
gM motion not recognized in ideaVim

### DISCUSSION THREAD

**Discussion by ogrish55** [JetBrains/ideavim#570] [Q&A] [answered]
Hello everyone. I've recently stumbled upon the gM motion in vim.

                                                        *gM*
gM                      Like "g0", but to halfway the text of the line.
                        With a count: to this percentage of text in the line.
                        Thus "10gM" is near the start of the text and "90gM"
                        is near the end of the text.

However the motion doesn't seem to work in ideaVim.
Does anyone have an idea how to make it work?

**Comment by citizenmatt** [comment, marked as answer]
Unfortunately, `gM` isn't currently implemented. You can vote for and track [VIM-2688](https://youtrack.jetbrains.com/issue/VIM-2688/Motion-gM-not-behaving-as-it-does-in-Vim), but the good news is that it's currently in progress. I have an implementation locally, as part of work I'm doing to support soft wraps. However, there is no ETA on when this will be available, as soft wraps is quite a large and tricky area to support (I am hoping it will be soon, though!)
